CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 0.28.6 -- 2020-06-3
```

Comment:
Please keep the changelog entries in the "unreleased section", I'll add the version header when I actually do the release.

Reply:
I'm sorry, i didn't understood how Changelog was managed.